test/powershell/Scripting/NativeExecution/NativeStreams.Tests.ps1

```diff
@@ -7,13 +7,10 @@ Describe "Native streams behavior with PowerShell" -Tags 'CI' {
 
         $error.Clear()
 
-        $command = [string]::Join('', @(
-            '[Console]::Error.Write(\"foo`n`nbar`n`nbaz\"); ',
-            '[Console]::Error.Write(\"middle\"); ',
-            '[Console]::Error.Write(\"foo`n`nbar`n`nbaz\")'
-        ))
-
-        $out = & $powershell -noprofile -command $command 2>&1
+        # because PowerShell now globs arguments given to native programs,
+        # we use --% so that '::' is not interpreted as a filesystem provider separator;
+        # but this also means that the command cannot be surrounded by quotes as they are not stripped
+        $out = & $powershell 2>&1 -noprofile -command --% [Console]::Error.Write(\"foo`n`nbar`n`nbaz\"); [Console]::Error.Write(\"middle\"); [Console]::Error.Write(\"foo`n`nbar`n`nbaz\")
```

Comment:
(By the way, if you do the non-sensical test of:

```powershell
~ |-/ dir '[Console]::Error.Write(\"foo`n`nbar`n`nbaz\");'                                                                                                
dir : Cannot find a provider with the name '[Console]'.
At line:1 char:1
+ dir '[Console]::Error.Write(\"foo`n`nbar`n`nbaz\");'
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: ([Console]:String) [Get-ChildItem], ProviderNotFoundException
    + FullyQualifiedErrorId : ProviderNotFound,Microsoft.PowerShell.Commands.GetChildItemCommand
 
```

the same problem happens due to globbing. This is just a weird scenario because we're passing non-path data to a native command, but now glob arguments to native commands; and in this case we don't want to do that, so use verbatim.)